Dockerfile

```diff
@@ -0,0 +1,75 @@
+FROM ubuntu:16.04
+
+# Docker remote API connection vars
+ENV DOCKER_CERT_PATH=/root/.supervisor/docker/
+ENV DOCKER_TLS_VERIFY=1
+ENV COMPOSE_HTTP_TIMEOUT=300
```

Comment:
I should have set this to `600` seconds for better reliability over cellphone based or international connections.